runtime/onert/core/src/ir/OperationDumper.cc

```diff
@@ -166,10 +166,10 @@ void OperationDumper::visit(const Fill &node)
 
 void OperationDumper::visit(const FullyConnected &node)
 {
-  std::string inputs =
-    "Weight(" + std::to_string(node.getInputs().at(FullyConnected::Input::WEIGHT).value()) +
-    ") Bias(" + std::to_string(node.getInputs().at(FullyConnected::Input::BIAS).value()) + ")";
-  dumpUnaryInputOp(node, inputs);
+  VERBOSE(LIR) << "* " << node.name() << std::endl;
+  VERBOSE(LIR) << "  - Inputs : Weight(" << node.getInputs().at(FullyConnected::Input::WEIGHT)
```

Comment:
Could you add FullyConnected::Input::INPUT as well?

Reply:
@ragmani As I wrote in https://github.com/Samsung/ONE/pull/12064#issuecomment-1818730616, I added `INPUT` that I've missed by mistake.